src/go/kreq-gen/go.mod

```diff
@@ -0,0 +1,8 @@
+module kafka-request-generator
+
+go 1.16
```

Comment:
We're currently using 1.18 fwiw

Reply:
This project uses whatevers on the system though. Does this value represent a min supported version?